packs/mssql/actions/execute.query.yaml

```diff
@@ -0,0 +1,33 @@
+---
+pack: mssql
+name: execute.query
+description: Queries MS SQL Server and writes the results to CSV. Returns list of paths to CSV files.
+enabled: true
+
+runner_type: python-script
+entry_point: mssql_runner.py
+
+parameters:
+  action:
+    default: execute_query
+    immutable: true
+    type: string
+  query_string:
+    description: Query to send to MS SQL Server.
+    required: true
+    type: string
+  params:
+    description: Parameters for interpolation in the query_string.
+    type: string
```

Comment:
I see, thanks for the clarification.

In this case I'm fine with making an exception and using a string for the parameter type here.